projects.go

```diff
@@ -367,6 +367,27 @@ func (s *ProjectsService) CreateProject(opt *CreateProjectOptions, options ...Op
 	return p, resp, err
 }
 
+// ShareWithGroupOptions represents options to share project with groups
+//
+type ShareWithGroupOptions struct {
+	ID          *int              `url:"id" json:"id"`
```

Comment:
`ID` is not an option, so it should not be in the options struct, but in the function signature.